stan/math/prim/fun/inv_sqrt.hpp

```diff
@@ -46,7 +46,7 @@ inline auto inv_sqrt(const T& x) {
  *
  * @tparam Derived derived type of x
  * @param x Matrix or matrix expression
- * @return Arc cosine of each variable in the container, in radians.
+ * @return 1 / sqrt of each value in x.
```

Comment:
[optional] I would say "The inverse square root ..." (same for the other instance in this file).

Reply:
Done.